api/utils/keys/yubikey.go

```diff
@@ -457,6 +461,13 @@ func (y *YubiKey) getPrivateKey(slot piv.Slot) (*PrivateKey, error) {
 		return nil, trace.Wrap(err)
 	}
 
+	// We don't yet support pin policies so we must return a user readable error in case
+	// they passed a mis-configured slot. Otherwise they will get a PIV auth error during signing.
+	// TODO(Joerger): remove this check once PIN prompt is supported.
+	if attestation.PINPolicy != piv.PINPolicyNever {
+		return nil, trace.NotImplemented(`PIN policy is not currently supported. Please generate a key with PIN policy "never"`)
```

Comment:
Can you add some information on how to generate the PIN with policy never?

Reply:
This is going to be removed very soon anyways, once https://github.com/gravitational/teleport/pull/31743 is merged.